charts/flink-tools/templates/FlinkCluster.yaml

```diff
@@ -5,6 +5,20 @@
 # You may obtain a copy of the License at
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
+{{- if .Values.enableExtraVolume }}
```

Comment:
As mentioned earlier, do not create the PVC in Helm.

Reply:
created separate sample files for PV and PVC